integration_test/feature_tracking/feature_tracking.go

```diff
@@ -0,0 +1,57 @@
+package feature_tracking
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/ops-agent/internal/set"
+	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
+)
+
+var errMissingFeatureTrackingMetrics = errors.New("missing feature tracking metrics")
+
+type MissingExpectedFeatureTrackingMetricsError struct {
+	Err                     error
+	MissingExpectedFeatures set.Set[FeatureTracking]
+}
+
+func (e *MissingExpectedFeatureTrackingMetricsError) Error() string {
+	return e.Err.Error() + ": " + fmt.Sprintf("%v", e.MissingExpectedFeatures)
+}
+
+type FeatureTracking struct {
+	Module  string
+	Feature string
+	Key     string
+	Value   string
+}
+
+type FeatureTrackingContainer struct {
+	Features []*FeatureTracking `yaml:"features"`
+}
+
+func AssertFeatureTrackingMetrics(series []*monitoringpb.TimeSeries, features []*FeatureTracking) error {
```

Comment:
This function (and the error type) should be moved to test code. I don't see a use case for it in the `feature_tracking` package.

Reply:
Changed file name to `feature_tracking_metadata`